onnxruntime/python/tools/transformers/fusion_embedlayer.py

```diff
@@ -98,7 +101,9 @@ def fuse(self, node, input_name_to_nodes, output_name_to_node):
                 return
             children = input_name_to_nodes[node.output[0]]
             children_types = sorted([child.op_type for child in children])
-            if children_types != ['MatMul', 'MatMul', 'MatMul', 'SkipLayerNormalization'] and children_types != ['MatMul', 'MatMul', 'MatMul', 'Shape', 'Shape', 'SkipLayerNormalization']:
+            if children_types != ['MatMul', 'MatMul', 'MatMul', 'SkipLayerNormalization'] and children_types != [
```

Comment:
Is there a line limit requirement?

Reply:
There is a [format tool](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/ReformatSourcePython.bat) that could deal with line limit automatically.